packages/modules/create-imodel/src/components/base-imodel/BaseIModel.scss

```diff
@@ -80,3 +75,8 @@
   height: 100%;
   justify-content: space-between;
 }
+
+.iac-input-element {
```

Comment:
Maybe too generic name. What if some other admin-components lib will need to have something similar?

Reply:
fixed